bin/cron.py

```diff
@@ -44,13 +47,39 @@
 MANAGE = str(ROOT_PATH / "manage.py")
 HEALTH_FILE_BASE = f"{DATA_PATH}/last-run"
 
+# The jobs are run every five minutes, so should all normally complete within that
+# timeframe. If they don't (eg, they've hung), we set a timeout just before then,
+# so that they are killed and we will try again.
+TIMEOUT_SECS = 290  # Just shy of five minutes.
+
+
+sentry_dsn = config("SENTRY_DSN")
```

Comment:
We can get this because we've loaded the configuration manager earlier on. I've verified it's possible in a local docker container, so I think it should be fine a deployed environment, _assuming_ that the relevant pod is allowe to make network calls out to Sentry, that is...

Reply:
Yeah. We know the config thing works. I don't see why it wouldn't be able to make network requests to Sentry if it can make them to all of the other data sources we load.

Reply:
Need to add `raise_error=False` to this call though, or it will throw an exception when `SENTRY_DSN` isn't in the environment.

Reply:
Good call - fixed